test/hipcub/test_utils.hpp

```diff
@@ -65,6 +65,59 @@ inline auto get_random_data(size_t size, T min, T max, int seed_value)
     return data;
 }
 
+#if defined(_WIN32) && defined(__clang__)
+#if 0
```

Comment:
Can we add a comment why we need these unused lines?

Reply:
Sorry, these were leftover lines.   Removed.